src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -2,4 +2,14 @@ package mate.academy
 
 class LetterRemover {
     // implement ...
+
+    companion object {
+        @JvmStatic
+        fun removeLetter(text: String): CharArray {
```

Comment:
The method name `removeLetter` suggests that it should remove a letter, but the implementation replaces 'r' with '!'. Consider renaming the method to something like `replaceLetter` to better reflect its functionality.